src/drt/src/gc/FlexGC_main.cpp

```diff
@@ -1866,6 +1866,8 @@ void FlexGCWorker::Impl::checkMetalShape_addPatch(gcPin* pin, int min_area)
     // detect what drNet has objects overlapping with the patch
     checkMetalShape_patchOwner_helper(patch.get(), dr_nets);
   }
+  if (!patch->hasNet())
+    return;
```

Comment:
When does it happen that a patch doesn't have a net?

Reply:
if checkMetalShape_patchOwner_helper(patch.get(), dr_nets) does not assign it to a drNet. It actually happens but It gets handeled later on when the drWorker commits the gc patches. It happens because the patch box sometimes does not touch the drNet at all. I am working in parallel to find why would that happen and I found it is  cause by a problem in the cornerspacing patching (fixed it in parallel but still testing it  on CIs)